principles/solid/liskvsustitution.md

```diff
@@ -1,14 +1,60 @@
 ## Liskov Substitution
 
-[TODO]
+We should be able to use a child class to replace a parent class without getting errors.
```

Comment:
The description of the principles is very simple. The principle should have the problem and how the principle solves it.

Reply:
Done!